### PR TITLE
Add email to the Avatar picker

### LINKED ISSUE #346: Quick Editor: Add email at the top
https://www.figma.com/design/xgXj4jig9m0DYfKjFq6swG/QE-Native-Experience?node-id=44-1094&t=6N8vBEajkfr2yJEe-0

### PR DESCRIPTION
**Pull request by pinarol**
Closes #346 

### Testing Steps

- SwiftUI demo app > avatar picker

Email should be visible at the top.

### CODE REVIEW

File: Sources/GravatarUI/SwiftUI/View+Additions.swift [hunk 1]
```diff
@@ -20,12 +20,13 @@ extension View {
...
-    func avatarPickerBorder(colorScheme: ColorScheme) -> some View {
+    func avatarPickerBorder(colorScheme: ColorScheme, borderWidth: CGFloat = 1) -> some View {
         self
             .shape(
                 RoundedRectangle(cornerRadius: 8),
                 borderColor: Color(UIColor.label).opacity(colorScheme == .dark ? 0.16 : 0.08),
-                borderWidth: 1
+                borderWidth: borderWidth
             )
+            .padding(.vertical, borderWidth) // to prevent borders from getting clipped
```
Comment (pinarol) [inline Sources/GravatarUI/SwiftUI/View+Additions.swift:30]: not directly related with this PR but I saw that the borders are getting clipped a bit, this fixes it.

File: Sources/GravatarUI/SwiftUI/AvatarPicker/AvatarPickerView.swift [hunk 2]
```diff
@@ -71,12 +76,23 @@ struct AvatarPickerView: View {
...
+        }
+    }
+
     private func header() -> some View {
         VStack(alignment: .leading) {
             Text("Avatars")
                 .font(.title2.weight(.bold))
             Text("Choose or upload your favorite avatar images and connect them to your email address.")
                 .font(.subheadline)
+                .foregroundColor(Color(UIColor.secondaryLabel))
```
Comment (pinarol) [inline Sources/GravatarUI/SwiftUI/AvatarPicker/AvatarPickerView.swift:95]: Not directly related with this PR but this is how it is in the designs and only a 1 line change so I wanted to include here.

**Review by etoledom — APPROVED** [review]
Looking good!

I would only ask to add an email in the xcode preview, but other than that, all good 🎉

**Comment by pinarol** [comment]
Thanks a lot. I addressed that. This PR is blocked by https://github.com/Automattic/Gravatar-SDK-iOS/pull/358 so it'd be great if you also review that one. @etoledom

**Pull request merged**